src/DataService/Media/Media.php

```diff
@@ -811,6 +811,12 @@ public function setCredits(array $credits)
 
     /**
      * Returns the streamingUrl of the default thumbnail for this Media.
+     *
+     * Note that the defaultThumbnailUrl may include dynamic URL substitution
```

Comment:
That's amazing, because I'm pretty sure that breaks the RFC for what a valid URL protocol is like! It's too bad they didn't go with protocoless URLs.

Reply:
Yeah :shrug: